services/getstream/followTopic.js

```diff
@@ -109,7 +130,6 @@ const removeTopicFromChatTab = async (token, topicName, userId) => {
   try {
     const name = topicName.toLowerCase();
     const client = new StreamChat(Environment.GETSTREAM_API_KEY, Environment.GETSTREAM_API_SECRET);
-    await client.connectUser({id: userId}, token);
     const channel = await client.channel('topics', `topic_${name}`);
 
     await channel.removeMembers([userId]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the removal of `client.connectUser` call might cause issues if the user is not already connected. Please ensure that the user connection is handled elsewhere in the code before this function is called. If not, you may need to reintroduce the `client.connectUser` call.

```diff
+    await client.connectUser({id: userId}, token);
     const channel = await client.channel('topics', `topic_${name}`);
 
     await channel.removeMembers([userId]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->